packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -62,13 +64,13 @@ Uploading...
       await this.botonicApiService.getMoreBots(bots, nextBots)
     }
     let bot = bots.filter(b => b.name === botName)[0]
-    if (bot == undefined) {
+    if (bot === undefined) {
```

Comment:
This looks really fishy to be honest... Generally `bot` should always return either some kind of "bot" object or undefined but you don't really check undefined like that ever. And here we actually only want this bot object. I'd suggest switching contents of if and else blocks and use `if (bot)`.